_posts/2018-10-07-how-to-use-elixir-ls-with-vim.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: How to use Elixir LS with Vim
+date: 2018-10-07 09:00:00 -04:00
+categories: post
+desc: Guide on how to use the Elixir LS language server with the Vim text editor. 
+permalink: /:categories/:year/:month/:day/:title/
+---
+
+### What is Elixir LS?
+
+[Elixir LS](https://github.com/JakeBecker/elixir-ls) by Jake Becker is the language server implementation for Elixir.
+
+### What is a language server?
+
+If you've been following the story of Visual Studio Code, there is a chance you've heard of another recent creation from Microsoft: the Language Server Protocol. 
```

Comment:
Maybe add a link to https://langserver.org/ or https://github.com/Microsoft/language-server-protocol ?

Reply:
Good idea, thought i did that.